astropy/coordinates/name_resolve.py

```diff
@@ -155,9 +160,9 @@ def get_icrs_coordinates(name, parse=False):
     exceptions = []
     for url in urls:
         try:
-            # Retrieve ascii name resolve data from CDS
-            resp = urllib.request.urlopen(url, timeout=data.conf.remote_timeout)
-            resp_data = resp.read()
+            resp_data = get_file_contents(
+                download_file(url, cache=cache, show_progress=False,
+                              timeout=data.conf.remote_timeout))
```

Comment:
```suggestion
                download_file(url, cache=cache, show_progress=False))
```

`timeout` setting here is redundant, as it is already the default behavior:

https://github.com/astropy/astropy/blob/986d5b74145c4cf6ee47c303d709832d199df24f/astropy/utils/data.py#L1138-L1139